packages/docs/src/styles/components/footer.scss

```diff
@@ -1,88 +1,35 @@
 .c-footer {
-  .ds-footer__container {
-    margin-inline: auto;
+  border-top: 2px solid $color-gray-lighter;
+  padding: 1.5rem 1.5rem 0;
```

Comment:
Can we match the spacing of the article element of `ds-u-sm-padding-x--6` for the footer?